som_account_invoice_pending/wizard/wizard_pay_invoice.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+from osv import osv
+from tools.translate import _
+from ast import literal_eval
+from oorq.decorators import job
+from tools import config
+
+class WizardPayInvoice(osv.osv_memory):
+
+    _name = "facturacio.pay.invoice"
+    _inherit = "facturacio.pay.invoice"
+
+    def action_pay_and_reconcile(self, cursor, uid, ids, context=None):
+        """Enviament d'un avís en donar per pagades factures amb un estat pendent concret
+        """
+        if not context:
+            context = {}
+
+        if isinstance(ids, list) or isinstance(ids, tuple):
+            ids = ids[0]
+
+        ir_md_obj = self.pool.get('ir.model.data')
+        fact_obj = self.pool.get('giscedata.facturacio.factura')
+        fact_id = context.get('active_id', False)
+        factura = fact_obj.browse(cursor, uid, fact_id)
+        tpv_journal = ir_md_obj.get_object_reference(cursor, uid,
+                      'som_account_invoice_pending', 'tpv_journal')[1]
+        wiz = self.browse(cursor, uid, ids)
+
+        cfg_obj = self.pool.get('res.config')
+        pay_alert_conf = cfg_obj.get(cursor, uid, 'pay_alert_pending_states_som', '[]')
+
+        if factura.pending_state.id in literal_eval(pay_alert_conf) and wiz.journal_id.id == tpv_journal:
+            self.mail_avis_cobraments(cursor, uid, ids, [fact_id], context)
+
+        super(WizardPayInvoice, self).action_pay_and_reconcile(cursor, uid, ids, context)
+
+        wiz = self.browse(cursor, uid, ids)
+        old_comment = factura.comment if factura.comment else ''
+        new_comment = wiz.comment + "\n"
```

Comment:
```suggestion
        new_comment = wiz.comment or '' + "\n"
```

sinó, quan no hi ha comentari dona error en fer un append de bool + string